### PR TITLE
Public Sector New Launch

### PR DESCRIPTION
**Pull request by Henrynunez112**
*Description*
Created the new covers on the public sector on the Government link in the Celor.org site

**Cover**
Screen Shot 2021-09-24 at 5 32 00 PM

**Public Sector**
Screen Shot 2021-09-24 at 5 33 09 PM

issue #326

### CODE REVIEW

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
I cant approve this. basing logic off the text of the title will make this way to brittle.

File: src/contentful/grid2-cells/Heading.tsx [hunk 1]
```diff
@@ -25,15 +25,22 @@ export function Heading(props: Props & HeadingContentType) {
   }
 
   const imageFile = props.image?.fields?.file
-
   return (
     <div css={css(rootCss, props.cssStyle, { gridColumn: `span ${props.span}` })}>
       {imageFile && (
+        props.title === "Public Sector on Celo" ?
```
Comment (aaronmgdr) [inline src/contentful/grid2-cells/Heading.tsx:31]: gotta fix this too

File: src/contentful/grid2-cells/Heading.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ interface Props {
 }
 
 export function Heading(props: Props & HeadingContentType) {
+  console.log(props, "<---- this is props")
```
Comment (aaronmgdr) [inline src/contentful/grid2-cells/Heading.tsx:14]: remove

File: src/contentful/grid2-cells/Heading.tsx [hunk 3]
```diff
@@ -25,15 +26,22 @@ export function Heading(props: Props & HeadingContentType) {
   }
 
   const imageFile = props.image?.fields?.file
-
   return (
     <div css={css(rootCss, props.cssStyle, { gridColumn: `span ${props.span}` })}>
       {imageFile && (
+        props.newIcon ?
```
Comment (aaronmgdr) [inline src/contentful/grid2-cells/Heading.tsx:32]: does newIcon exist here?

Comment (Henrynunez112) [reply src/contentful/grid2-cells/Heading.tsx:32]: yeah

File: src/contentful/grid2-cells/Blurb.tsx [hunk 4]
```diff
@@ -45,10 +46,19 @@ const renderWhiteParagraphWithRow = { ...renderWhiteParagraph, ...embeddable }
...
+  let width: number
+  let height: number
+  if(props.newIcon){
+  width = props.isNaturalSize ? image?.details?.image?.width : 48
+  height = props.isNaturalSize ? image?.details?.image?.height : 48
+}else{
+  width = props.isNaturalSize ? image?.details?.image?.width : 100
+  height = props.isNaturalSize ? image?.details?.image?.height : 100
+
+}
```
Comment (aaronmgdr) [inline src/contentful/grid2-cells/Blurb.tsx:61]: can you format this?

Comment (aaronmgdr) [reply src/contentful/grid2-cells/Blurb.tsx:61]: super nit i know

Comment (Henrynunez112) [reply src/contentful/grid2-cells/Blurb.tsx]: yessir

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**